src/layout.rs

```diff
@@ -22,8 +22,17 @@ pub enum SizingMode {
     InherentSize,
 }
 
+/// A struct containing both the size of a node and it's first baseline in each dimension (if it has any)
```

Comment:
This could use a simple definition of what a "baseline" is.

Reply:
Fixed